### DISCUSSION TITLE
Add support for different type of Typed Schemas for Autoform (Valibot, Yup)

### DISCUSSION THREAD

**Discussion by affemaen** [unovue/shadcn-vue#523] [Ideas]
I'm currently using Valibot in my project and noticed the new component called Auto form. It would be really nice to be able to use Autoform with valibot instead of Zod.

VeeValidate supports Zod, Valibot and Yup for typed schemas [Typed Schemas](https://vee-validate.logaretm.com/v4/guide/composition-api/typed-schema). Are there any plans to add support for Valibot and Yup?

**Comment by zernonia** [comment]
Thanks for your suggestion @affemaen . Yup we acknowledge that VeeValidate supports Zod validation. However we chose to support Zod at this moment because of their strongly-typed schema, and it works pretty well with VeeValidate. I know Yup doesn't offer strong typing, and I have yet to try out Valibot.
